applications/rogue/main.tf

```diff
@@ -81,20 +87,41 @@ resource "random_id" "app_key" {
 }
 
 locals {
+  # Gambit only has QA & Production environments.
+  gambit_env = var.environment == "development" ? "qa" : var.environment
```

Comment:
Context: As far as I know, I can't cleanly make the data resources (to read the username/password from SSM) optional, and so it's easiest to just plop the QA credentials onto the dev environment & disable via feature flag.

Reply:
Is the feature flag happening in this file? I would have thought `DS_ENABLE_GAMBIT_RELAY` would be false for dev?

Reply:
OH I see, you only added the Gambit url to prod and QA yeah?

Reply:
Yep!